docs/core/tools/dotnet-new-sdk-templates.md

```diff
@@ -21,6 +21,30 @@ Each template may have additional options available. To show the additional opti
 In case the template supports multiple languages, this command will show help for the template in the default language. By combining it with the `--language` option, you can see the help for other languages: `dotnet new console --help --language F#`.
 The templates that ship with the .NET SDK have the following additional options:
 
+## `buildprops`
```

Comment:
Since all of these sections are really about the options for each template, I feel like each template heading should be an H3, not an H2, to make them subheadings of `## Template options`.

Reply:
I'll address it in a separate PR and you can see what you think.